ui/src/shared/utils/resourceToTemplate.ts

```diff
@@ -96,8 +96,87 @@ export const taskToTemplate = (
   return template
 }
 
-// Todo: update template to match task format
-export const dashboardToTemplate = (dashboard: Dashboard) => ({
-  meta: {chronografVersion: '2.0'},
-  dashboard,
+const viewToIncluded = (view: View) => ({
+  type: TemplateType.View,
+  id: view.id,
+  attributes: view
+})
+const viewToRelationship = (view: View) => ({
+  type: TemplateType.View, id: view.id
 })
+
+
+const cellToIncluded = (cell: Cell, views: View[]) =>  {
+  const cellView = views.find(v => v.id === cell.id)
+  const viewRelationship = viewToRelationship(cellView)
+
+  return {
+    id: cell.id,
+    type: TemplateType.Cell,
+    attributes: cell,
+    relationships: {
+      [TemplateType.View]: {
+        data: viewRelationship
+      }
+    }
+  }
+}
+
+const cellToRelationship = (cell: Cell) => ({
+  type: TemplateType.Cell, id: cell.id
+})
+
+// Todo: update template to match task format
+export const dashboardToTemplate = (
+  dashboard: Dashboard,
+  views: View[],
```

Comment:
I think it would be appropriate for the client to worry about getting views instead of expecting to be passed.  Now that I think of it, this should be a function that takes just a dashboard id and does everything else itself. Let's get this in as is, but I'll open an issue to refactor.